app/code/Magento/MediaStorage/etc/config.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0"?>
+<!--
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+-->
+<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:module:Magento_Store:etc/config.xsd">
+    <default>
+        <system>
+            <media_configuration>
```

Comment:
This seems to be `uploader_configuration`, isn't it?

Currently these changes look flawed from modularity perspective. Configuration added to `MediaStorage` module but no logic in it, some code added to framework. Please arrange your changes properly.